app/controllers/users/sessions_controller.rb

```diff
@@ -16,6 +16,23 @@ class Users::SessionsController < Devise::SessionsController
   #   super
   # end
 
+  # Singing in with authentication token (needed when signing in automatically
+  # from another website)
+  def auth_token_create
+    user = User.find_by_email(params[:user_email])
+    if user.authentication_token == params[:user_token][0..-2]
+      sign_in(:user, user)
```

Comment:
I believe that we want this token to be a single-use. Therefore, after user is successfully signed in via this token, we need to refresh it (create a new one). This can simply be done by setting it to `nil` ([docs](https://github.com/gonzalo-bulnes/simple_token_authentication#tokens-generation)). So I propose that after `sign_in(:user, user)` line, add following:

```ruby
sign_in(:user, user)

# This will cause new token to be generated
user.update(authentication_token: nil)
```